vue-components/nightwatch.config.js

```diff
@@ -1,8 +1,65 @@
+const build = `Nightwatch build-${process.env.DATE}`;
+
 /* eslint-disable @typescript-eslint/camelcase */
 module.exports = {
+	src_folders: [ 'tests/e2e/specs' ],
+	page_objects_path: 'tests/e2e/page-objects',
+	custom_commands_path: 'tests/e2e/custom-commands',
+
+	test_workers: {
+		enabled: true,
+		workers: 'auto',
+	},
+
 	test_settings: {
 		default: {
-			launch_url: 'http://localhost:6005/',
+			launch_url: 'https://wmde.github.io/wikit/vue-components/',
```

Comment:
Temp hard-coded to the production storybook for testing purposes.
In the final solution this will ideally be dynamic and take the URL chromatic generates per branch.